man/mono.1

```diff
@@ -118,11 +113,16 @@ object file.
 If specified, forces the generated AOT files to be bound to the
 runtime version of the compiling Mono.   This will prevent the AOT
 files from being consumed by a different Mono runtime.
+.TP
+.I bitcode
```

Comment:
The bitcode output is not the defining characteristic of this mode so describing it that way is misguided.

Other modes produce a bitcode file when using llvm so this explanation is confusing.